components/director/internal/model/labledef_test.go

```diff
@@ -63,3 +63,51 @@ func TestValidateLabelDef(t *testing.T) {
 	})
 
 }
+
+func TestValidateForUpdateLabelDef(t *testing.T) {
+	t.Run("valid input when schema not provided", func(t *testing.T) {
+		// GIVEN
```

Comment:
do we really need to have 2 methods: Validate and ValidateForUpdate()? - right now we have 2 very similar methods and we have to copy tests for them. If in Update method we fetch definition, get id for it and set ti to the model then we can probably remove ValidateForUpdate

Reply:
added TODO